packages/prism-theme-odyssey/src/index.js

```diff
@@ -0,0 +1,3 @@
+import '@okta/odyssey'
+import 'prismjs'
```

Comment:
Please make sure these are called out as `dependencies` in our `package.json`!

_Originally posted by @jmelberg-okta in https://github.com/okta/odyssey/pull/714#discussion_r509797256_

Reply:
Completed as part of this PR.